src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,20 @@
-export const TodoFilter = () => (
+import React from 'react';
+
+type Props = {
+  handlerSelect: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  handlerInput: (event: React.ChangeEvent<HTMLInputElement> | string) => void;
```

Comment:
The `handlerInput` prop is used for both handling input changes and clearing the input. Consider separating these concerns by using distinct functions for each action to improve clarity.